src/main/java/de/cronn/validation_files_diff/impl/ValidationFileAutomaticRenameFactory.java

```diff
@@ -0,0 +1,48 @@
+package de.cronn.validation_files_diff.impl;
+
+import com.intellij.psi.PsiClass;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiMethod;
+import com.intellij.refactoring.rename.naming.AutomaticRenamer;
+import com.intellij.refactoring.rename.naming.AutomaticRenamerFactory;
+import com.intellij.usageView.UsageInfo;
+import de.cronn.validation_files_diff.ValidationDiffApplicationOptionsProvider;
+import de.cronn.validation_files_diff.helper.PsiElementValidationFileFinder;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collection;
+
+public class ValidationFileAutomaticRenameFactory implements AutomaticRenamerFactory {
+
+	public static final String OPTION_NAME = "Rename Validation-Files";
```

Comment:
Access can be 'private'

Reply:
Done in a27874cb10b3c30749af1fdc390e0e9ed07d9986